src/components/Photos.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Photo from './Photo';
+import styles from './Photos.css';
+
+
+function Photos({ urls }) {
+  const photoList = urls.map(url => {
+    return <li key={name}><Photo url={url} /></li>;
```

Comment:
```suggestion
    return <li key={url}><Photo url={url} /></li>;
```
In this case, you can use the url as the key because it is a unique value.